app/backend/src/couchers/servicers/donations.py

```diff
@@ -24,7 +24,7 @@ def _create_stripe_customer(session: Session, user: User) -> None:
     customer = stripe.Customer.create(
         email=user.email,
         # metadata allows us to store arbitrary metadata for ourselves
-        metadata={"user_id": user.id},
+        metadata={"user_id": user.id},  # type: ignore[dict-item]
```

Comment:
@aapeliv stripe expects it to be dict[str, str], but it seems that we don't really use it anymore?

Reply:
We need the info in stripe just to have it there